server/django_server/permissions.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.permissions import BasePermission
+from user_account.models import CustomUser
+
+
+class isSystemAdmin(BasePermission):
+    message = "You must be a System Admin to do this operation"
+
+    def has_permission(self, request, view):
+        """
+        Overriding default has_permission method in order to add Custom permissions to our views
+        This can be used either inside the permission_class directly or you can call it from other permission files
+        :param request:
+        :param view:
+        :return: True/False : Whether the user is a SysAdmin or Not
+        """
+        user = CustomUser.objects.get(user_name=request.user)
+        if user.role == 'SA':
+            return True
+        else:
+            return False
+
+
+class isInventoryManager(BasePermission):
+    message = "You must be an Inventory Manager to do this operation"
+
+    def has_permission(self, request, view):
+        """
+        Overriding default has_permission method in order to add Custom permissions to our views
+        This can be used either inside the permission_class directly or you can call it from other permission files
+        :param request:
+        :param view:
+        :return: True/False : Whether the user is a Inventory Manager or Not
+        """
+        user = CustomUser.objects.get(user_name=request.user)
+        if user.role == 'IM':
+            return True
+        else:
+            return False
```

Comment:
You can return the expression itself to save loc
return user.role == 'IM'

Reply:
Yes, I'll fix it.